test-files/cmd/slow/ConflateCmdStatsAllDataTypesTest.sh

```diff
@@ -5,7 +5,7 @@ mkdir -p test-output/cmd/ConflateCmdStatsTest
 STATS_OUT=test-output/cmd/ConflateCmdStatsTest/all-data-types-out
 SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
-hoot conflate -D writer.include.debug.tags=true test-files/conflate/unified/AllDataTypesA.osm test-files/conflate/unified/AllDataTypesB.osm $STATS_OUT.osm --stats > $STATS_OUT
+hoot conflate -D hash.seed.zero=true -D hash.seed.zero=true -D writer.include.debug.tags=true test-files/conflate/unified/AllDataTypesA.osm test-files/conflate/unified/AllDataTypesB.osm $STATS_OUT.osm --stats > $STATS_OUT
```

Comment:
This one is doubled up but creating a config file for testing should fix this also.

Reply:
Fixed